src/openrct2/actions/LoadOrQuitAction.cpp

```diff
@@ -18,6 +18,12 @@ LoadOrQuitAction::LoadOrQuitAction(LoadOrQuitModes mode, PromptMode savePromptMo
 {
 }
 
+void LoadOrQuitAction::AcceptParameters(GameActionParameterVisitor& visitor)
```

Comment:
I'm not sure this should be exposed in the scripting api.

Reply:
Why not?
You mentioned potential crashes (in another PR) but I tested this action a lot and it did not seem to have any problems.

Reply:
It resets the plugin when called (unless they are persistant plugins). Would plugin authors need this sort of thing?

Reply:
Sure, for a "Hardcore" mode that immediately quits the game without saving when you drop below 500 park rating.

Tbh I don't really see the point in arguing if a feature may be needed or not, _unless there is a good reason against that feature_. And as I said in other discussions before, I do not think that "it enables the potential for bad user experience" is a good reason, because there are already enough API features that do this.

Reply:
But I think this action would be among those where I would care least about if they don't make the cut.